framework/components/ecl-expandables/_expandables.scss

```diff
@@ -1,45 +1,4 @@
-/*
-Expandable
-*/
-
-.expandable__content {
```

Comment:
Let me explain :)

What the DTT called [expandable](https://ec-europa.github.io/ec-europa-theme/section-expandable.html#kssref-expandable-1) is, in fact, nothing more than an accordion, right? The problem is that they also have [expandable links](https://ec-europa.github.io/ec-europa-theme/section-expandable.html#kssref-expandable-2) or [button collapse](https://ec-europa.github.io/ec-europa-theme/section-buttons.html#kssref-buttons-btn-collapse)... This is confusing. 

What I'm willing to achieve here is to simplify the definition of "expandable" ("expandable" and "collapsible" are the same thing, only the initial state is different). Put it simply, an expandable component is a container wrapping content which expands or collapses when you click on a lever (a button for example). That's all. "Button collapse", "expandable links", I don't see any difference. They are just special cases of the "expandable behavior". I will add variants for them, so there will be some CSS rules to be added, but the "ecl-expandable" CSS component won't exist because it is an abstract (or virtual) component (see it as an "interface" in OOP 😄)